packages/slice-machine/src/features/onboarding/OnboardingProgressStepper.tsx

```diff
@@ -33,37 +35,46 @@ export const OnboardingProgressStepper = () => {
         isOpen={isDialogOpen}
         onClose={() => setDialogOpen(false)}
       />
-      <DropdownMenu>
+      <DropdownMenu open={isListOpen} onOpenChange={setListOpen}>
         <DropdownMenuTrigger>
           <Button
             color="grey"
             sx={{ width: "100%" }}
             renderEndIcon={EndCtaIcon}
+            onMouseEnter={() => setListOpen(true)}
           >
             {completedStepCount > 0 ? "Continue" : "Start"}
           </Button>
         </DropdownMenuTrigger>
-        <DropdownMenuContent align="start">
-          <DropdownMenuLabel>Progress</DropdownMenuLabel>
-          {steps.map((step) => {
-            const isCompleted = isStepComplete(step.id);
+        <DropdownMenuContent align="center" sideOffset={-56} minWidth={256}>
```

Comment:
I think the menu can be better aligned with the card. I see that this uses arbitrary numbers to position itself, which doesn't seem like a good way.

Reply:
yeah, i agree, i missed this 👍

Reply:
Honestly I think it's just fine like that: the dropdown content is positioned against the dropdown trigger. Now, calculating distance to the card (which is outside of this component) would add some complexity that for me is not necessary here. Should the design change, some change in code would be necessary anyways and then updating the offset value is no effort at all.

Reply:
For me it's ok if we leave it like this. I don't think the complexity is worth it. I would leave a comment there explaining why it's 56 and what to check if it changes in the future.

Reply:
@jomifepe I understand that using magic numbers is not a good practice, my bad that I didn't add any comment on that! 
Still I think it is just better to leave it (but with added comment) than adding more complexity (which probably also would need some explanation in the comments)

Reply:
Still think that it was worth trying out my suggestion. Your comment didn't clarify anything regarding the number 😅 `50` is what? Most likely `36px (button height) + 16px (padding) + 2px (default gap for radix dropdown menu)`.